powerVCF.psm1

```diff
@@ -3715,7 +3727,7 @@ Function Validate-EdgeClusterSpec
     checkVCFToken # Calls the CheckVCFToken function to validate the access token and refresh if necessary
   $uri = "https://$sddcManager/v1/edge-clusters/validations"
   Try {
-    $response = Invoke-RestMethod -Method POST -URI $uri -ContentType application/json -headers $headers -body $json
+    $response = Invoke-RestMethod -Method POST -ContentType "application/json" -URI $uri -Headers $headers -body $ConfigJson -SkipCertificateCheck
```

Comment:
SkipCertificateCheck will not work with non core PowerShell versions. There is code in the module to handle untrusted certificates.

Reply:
Jason seen comment from Brian regarding the SkipCertificateCheck this needs to be removed